experimental/dds/tree-graphql/.eslintrc.js

```diff
@@ -17,15 +17,6 @@ module.exports = {
 				avoidEscape: true,
 			},
 		],
-		// Intentionally not unifying signatures can enable more scoped API documentation and a better developer experience, which accounts
```

Comment:
These rules no longer appear in our base config, and so no longer need to be overridden.